component-library/src/app/components/interactive-demo/interactive-demo.component.html

```diff
@@ -10,7 +10,11 @@
   <div class="toggles-accordion">
     <app-accordian-container [config]="accordionConfig">
       <div class="toggles">
-        <ng-content select="[toggles]"></ng-content></div
-    ></app-accordian-container>
+        <ng-content select="[toggles]"></ng-content>
+      </div>
+      <div class="checkboxes">
+        <ng-content select="[checkboxes]"></ng-content>
```

Comment:
This was not included deliberately. Wrap your radios and checkboxes in a div and project them as one